src/main/scripts/src/app/workspace/organization/organization-settings.component.html

```diff
@@ -1,14 +1,71 @@
-<ul class="nav nav-tabs">
-  <li role="presentation" class="nav-item">
-    <a routerLink="settings" routerLinkActive="active" class="nav-link">
-      Basic Settings
-    </a>
-  </li>
-  <li role="presentation" class="nav-item">
+<div class="container-fluid">
+  <div class="row">
+    <div class="col"></div>
+    <div class="whole col-sm-9 col-md-10 col-lg-11" [style.opacity]="initialized() ? '1' : '0'">
+      <div class="d-flex flex-row justify-content-start">
+        <a class="fa fa-arrow-left " [routerLink]="['/workspace']  "></a>
+        <i class="color-class fa-fw {{ organization.icon }} organization-icon " [style.color]="organization.color"
+           data-toggle="dropdown"></i>
+        <div class="dropdown-menu">
+          <color-picker class="dropdown-item" [(color)]="organization.color"
+                        (selectedChange)="updateOrganization()"></color-picker>
+          <div class="dropdown-divider"></div>
+          <icon-picker class="dropdown-item" [(icon)]="organization.icon"
+                       (selectedChange)="updateOrganization()"></icon-picker>
+        </div>
+
+        <input class="color-class name" [(ngModel)]="organization.name" (blur)="updateOrganization()"
+               [style.color]="organization.color" i18n-title="@@header.organization.name"
+               title="Edit name of the organization">
+
+        <div class="float-right document-count color-class ml-auto">
+          <a i18n-title="@@organization.projects.count" class="count" title="Number of projects in this organization"
+             [style.color]="organization.color" [routerLink]="['/workspace']">
+            <span>{{ projectsCount }}</span>
+            <i class="far fa-file" aria-hidden="true"></i>
+          </a>
+
+        </div>
+
+      </div>
+
+      <div class="d-flex flex-row justify-content-start">
+
+        <input class="color-class code" [(ngModel)]="organization.code" (blur)="updateOrganizationCode()"
+               [style.color]="organization.color" i18n-title="@@header.organization.code"
+               title="Edit code of the organization">
+
+        <i class="fa fa-times pull-right text-danger icon ml-auto" aria-hidden="true"
+           i18n-title="@@header.organization.remove" title="Remove organization"
+           *ngIf="organizationCode" (click)="onDelete()"></i>
+
+      </div>
+
+      <div class="description d-inline-flex py-3">
```

Comment:
It would be better if the description spanned across the whole screen width.
`<div class="description d-inline-flex py-3">` -> `<div class="description d-flex py-3">`
`<div class="text" [attr.contenteditable]="organizationDescriptionEditable"....` -> `<div class="text flex-grow" [attr.contenteditable]="organizationDescriptionEditable"....`